### PR TITLE
Currency: Updated build script and build_info.json for LightGBM 

### PR DESCRIPTION
**Pull request by aastha-sharma2**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Have you validated script on UBI 9 container
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: l/lightGBM/lightGBM_ubi_9.3.sh [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        echo "installing scipy"
+        git clone https://github.com/scipy/scipy.git
+        cd scipy/
+        git checkout v1.10.1
+        git submodule update --init
+        pip install Cython==0.29.37 'numpy<1.23' 'setuptools<60.0' pybind11 pytest pythran  wheel
+        pip install Cython setuptools pybind11 pytest pythran  wheel numpy==1.19.5
+        ln -s /usr/lib64/atlas/libtatlas.so.3 /usr/lib64/atlas/libtatlas.so
+        ln -s /usr/lib64/libopenblas.so.0 /usr/lib64/libopenblas.so
+        python3 setup.py build
```
Comment (shubham-dayma-ibm) [inline l/lightGBM/lightGBM_ubi_9.3.sh:50]: Why building instructions?

**Review by shubham-dayma-ibm — CHANGES_REQUESTED** [review]
PR status is Draft, still reviewed it. Also missing license file

File: l/lightGBM/lightGBM_ubi_9.3.sh [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+pip install setuptools numpy==1.19.5 pandas==1.4.2 build joblib psutil pillow matplotlib
+
+#Install
+# Set the path to the python-package directory
+cd python-package
+if ! (python3 setup.py install) ; then
+    echo "------------------$PACKAGE_NAME:Install_fails-------------------------------------"
+    echo "$PACKAGE_URL $PACKAGE_NAME"
+    echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | GitHub | Fail | Install_Fails"
+    exit
```
Comment (shubham-dayma-ibm) [inline l/lightGBM/lightGBM_ubi_9.3.sh:78]: It should be `exit 1`

**Review by shubham-dayma-ibm — CHANGES_REQUESTED** [review]

File: l/lightGBM/lightGBM_ubi_9.3.sh (hunk 2 above)
Comment (aastha-sharma2) [reply l/lightGBM/lightGBM_ubi_9.3.sh]: > It should be `exit 1`

Updated.

**Review by nikhil-kalbande — APPROVED** [review]

**Pull request merged**